__tests__/settings.ts

```diff
@@ -239,6 +241,19 @@ describe('mutation updateUserSettings', () => {
     expect(res.data).toMatchSnapshot();
   });
 
+  it('should update selected algo by user for comments section', async () => {
+    loggedUser = '1';
+
+    const repo = con.getRepository(Settings);
+    const settings = await repo.save(repo.create({ userId: '1' }));
+    expect(settings.sortCommentsBy).toEqual('oldest');
+
+    const res = await client.mutate(MUTATION, {
+      variables: { data: { sortCommentsBy: SortCommentsBy.Newest } },
+    });
+    expect(res.data.updateUserSettings.sortCommentsBy).toEqual('newest');
```

Comment:
Test that the settings in the database matches as well?

Reply:
Updated 🚀